src/lib/support/PersistedCounter.h

```diff
@@ -61,7 +61,7 @@ class PersistedCounter : public MonotonicallyIncreasingCounter<T>
 {
 public:
     PersistedCounter() : mKey(StorageKeyName::Uninitialized()) {}
-    ~PersistedCounter() override {}
+    virtual ~PersistedCounter() override {}
```

Comment:
```suggestion
   ~PersistedCounter() override {}
```
override implies virtual, and we should be using one or the other but not both.